### PR TITLE
Adding `fractal-server`

### PR DESCRIPTION
**Pull request by tcompa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by tcompa** [comment]
I triggered a rebuild as I think the previous failure was due to https://github.com/conda-forge/staged-recipes/issues/24113.

**Comment by tcompa** [comment]
@conda-forge/help-python, ready for review!

File: recipes/fractal-server/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - clusterfutures >=0.5.0,<0.6.0
+    - cloudpickle >=2.2.1<2.3.0
+    - asyncpg >=0.27.0,<0.28.0
+    - psycopg2 >=2.9.5,<3.0.0
+    - gunicorn >=20.1.0,<21.0.0
+
+test:
+  imports:
+    - fractal_server
+  commands:
```
Comment (synapticarbors) [inline recipes/fractal-server/meta.yaml:47]: Could you test the cli under the test/commands section? Just `- fractalctl --help` or similar.

Comment (tcompa) [reply recipes/fractal-server/meta.yaml:47]: Of course, thanks! I fixed it with https://github.com/conda-forge/staged-recipes/pull/24119/commits/3e443d725ced7f3d422fc6018ce1508be5af4153.

**Comment by tcompa** [comment]
I implemented the requested change.
@conda-forge/help-python PR ready for review. Thanks!

**Comment by synapticarbors** [comment]
It looks like there are conflicts. I'm not sure why given the two different branches used for `fractal-client` vs `fractal-server`, but maybe something is going on that I don't have visibility into.

**Comment by tcompa** [comment]
No problem, I can create a fresh fork and re-open a new PR.

**Comment by synapticarbors** [comment]
Just ping me on the new PR and I'll merge it

**Comment by tcompa** [comment]
> Just ping me on the new PR and I'll merge it

@synapticarbors here it is https://github.com/conda-forge/staged-recipes/pull/24144, thanks!

**Pull request closed without merging**